dictionaries/html/src/html.txt

```diff
@@ -3002,6 +3006,7 @@ vsupnE
 Vvdash
 vzigzag
 wbr
+wcag
```

Comment:
Guessing this is for Web Content Accessibility Guidelines. Also not something that really needs to be in the HTML dictionary
```suggestion
WCAG
```

Reply:
A lowercase will match title case and fully uppercase words.

- `wcag` -> `wcag` ok, `Wcag` ok, `WCAG` ok
- `WCAG` -> `wcag` no, `Wcag` no, `WCAG` ok

Reply:
Its only really valid as all uppercase though, since it's an acronym